package/MDAnalysis/analysis/rms.py

```diff
@@ -528,8 +528,7 @@ def _single_frame(self):
             self._ts.positions[:] -= mobile_com
 
             # R acts to the left & is broadcasted N times.
-            self._ts.positions[:,:] = (self._mobile_coordinates64[:] *
-                                       self._R)
+            self._ts.positions[:, :] = self._ts.positions * self._R
```

Comment:
Well... I just updated our example with backtics as in numpy. Feel free to start a discussion :-).